portfolio/src/main/java/com/google/sps/servlets/JSONServlet.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Google LLC
```

Comment:
And do you still need this file, or can this be deleted in favor of the comment servlet?